### PR TITLE
Fix prompt for lambda role arn

### PR DESCRIPTION
**Pull request by hsubbaraj-spiral**
## Describe your changes and why you are making these changes
Fixes prompt for lambda role arn
## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const LambdaDialog: React.FC<Props> = ({ onUpdateField, value }) => {
         spellCheck={false}
         required={true}
         label="Lambda Role ARN"
-        description="Region in which to run Lambda functions."
+        description="ARN for Lambda executor role."
```
Comment (kenxu95) [inline src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx:23]: Is there a guide we could link to? Either ours or Amazons? I see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console

**Review by kenxu95 — APPROVED** [review]

File: src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx (hunk 1 above)
Comment (kenxu95) [reply src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx:23]: or maybe its https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html

Comment (hsubbaraj-spiral) [reply src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx:23]: Yea either our docs at https://github.com/aqueducthq/gitbook/blob/main/integrations/adding-an-integration/connecting-to-aws-lambda.md or the amazon link here: https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html#permissions-executionrole-console should work. I'm unsure why the documentation page isn't appearing directly on gitbook, i'll look into that.

**Comment by vsreekanti** [comment]
@hsubbaraj-spiral, what's the status of this PR?

**Pull request merged**